### DISCUSSION TITLE
[Unstable] Detonator's light is set to cast shadows.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14710] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Detonator & Timed-Detonator's light component is set to shadow casting, unintentional?

### Reproduction steps

![image](https://github.com/user-attachments/assets/a0d77be5-f4b2-453c-8ffa-5b3d3477c2fd)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested simple xml fix for this. No issues found.
Closing discussion as resolved.
